### DISCUSSION TITLE
Don't know what to do in 2.7.9 - many different How-Tos, nothing is working

### DISCUSSION THREAD

**Discussion by dhbdev** [logancyang/obsidian-copilot#1177] [Q&A]
Hi,

I'm confused because I tested plenty of different combinations. I always get a connection error with lm-studio. In the developer logs, I see the content of the message, but nothing is in the Obsidian chat.

Please provide a step-by-step tutorial which works because as a newbie, some information I do not have. And I'm not a developer, only a beginner user. The lm-studio setup seems to work fine.

I installed the following local LLM in lm-studio:
Mistral-7B-Instruct-v0.3-GGUF/Mistral-7B-Instruct-v0.3-Q4_K_M.gguf
mlx-community/Llama-3.2-3B-Instruct-4bit

Thanks Andreas

**Comment by logancyang** [comment]
Hi @astoeffer, have you checked the documentation? https://www.obsidiancopilot.com/en/docs

LM studio is working fine for me
<img width="1005" alt="SCR-20241216-kmst" src="https://github.com/user-attachments/assets/4b7c3897-d7cc-43f8-91ab-6ae63671a816" />
